src/main.c

```diff
@@ -272,6 +273,7 @@ CF_ABS("tcp_keepalive", CF_INT, cf_tcp_keepalive, 0, "1"),
 CF_ABS("tcp_keepcnt", CF_INT, cf_tcp_keepcnt, 0, "0"),
 CF_ABS("tcp_keepidle", CF_INT, cf_tcp_keepidle, 0, "0"),
 CF_ABS("tcp_keepintvl", CF_INT, cf_tcp_keepintvl, 0, "0"),
+CF_ABS("tcp_user_timeout", CF_INT, cf_tcp_user_timeout, 0, "0"),
```

Comment:
Not sure if this should have a default here?

Reply:
I think the logic you have is OK: By default, it's 0 and you leave the kernel setting alone. Only when you set it to non-zero it will get activated.

Reply:
Sounds good to me 👍